.github/workflows/gatk-nightly-docker.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and Upload GATK Nightly Docker Image
+on:
+  schedule:
+    # trigger the job the image at 1am est every night (UTC 20:00)
+    # NOTE: by default schedule:cron jobs pull from the repo default branch which makes this the master branch of gatk
+    - cron: '0 20 * * *'
+
+jobs:
+  buildDocker:
+    name: build docker images
+    runs-on: ubuntu-latest
```

Comment:
I hadn't thought about this before, but we can probably save 20 seconds by using alpine linux instead of ubuntu.  Kind of useless savings though.